sbin/pfmon

```diff
@@ -391,7 +411,7 @@ run all runtasks
 sub runtasks {
     my $mask = POSIX::SigSet->new(POSIX::SIGCHLD());
     sigprocmask(SIG_BLOCK,$mask);
-    while(@REGISTERED_TASKS) {
+    while(@REGISTERED_TASKS && $process) {
```

Comment:
From my POV having sleepy workers is really just having useless process... It is really more easy to tell that pfmon is running here or there by looking at the workers.
And spawning is pretty instantanious on "failover"